test/conformance/CMakeLists.txt

```diff
@@ -8,9 +8,18 @@ set(UR_CONFORMANCE_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 function(add_test_adapter name adapter)
     set(TEST_TARGET_NAME test-${name})
     set(TEST_NAME ${name}-${adapter})
+
     add_test(NAME ${TEST_NAME}
-        COMMAND ${TEST_TARGET_NAME}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+        COMMAND ${CMAKE_COMMAND}
+        -D TEST_FILE=${Python3_EXECUTABLE}
+        -D TEST_ARGS="${UR_CONFORMANCE_TEST_DIR}/cts_exe.py --test_command ${CMAKE_BINARY_DIR}/bin/${TEST_TARGET_NAME}"
```

Comment:
`${CMAKE_CURRENT_BINARY_DIR}` instead of `${CMAKE_BINARY_DIR}/bin` ?

and actually, since you're using WORKING_DIRECTORY you could omit this full path and just use `./${TEST_TARGET_NAME}`...?

Reply:
huh, interesting, thx for straightening me out